### DISCUSSION TITLE
bot现在是否还能使用？

### DISCUSSION THREAD

**Discussion by lTinchl** [azmiao/uma_plugin#68] [Q&A] [answered]
我看到go-cqhttp好像不能用了，bot发不出消息来了

**Comment by azmiao** [comment]
如果你还能登录8.9.80及以下版本，go-cq配合qsign还是能用的，如果bot登不了旧版了就只能换其他方案了，比如官方bot api或者其他什么的，具体可以去查查

**Reply by lTinchl** [reply]
大佬的txlib选的什么版本？

**Reply by azmiao** [reply]
8.9.70，我bot比较多，一号bot现在只能登9.0.0+了，现在换二号上岗了2333

**Reply by lTinchl** [reply]
坏事了，qsign没windows sever的本地部署教程了

**Reply by azmiao** [reply]
https://github.com/rhwong/qsign-onekey

**Reply by lTinchl** [reply]
[2024-03-01 11:42:34,984 hoshino] INFO: 【马娘插件】启动时检查更新...
[2024-03-01 11:42:35,155 hoshino] INFO: ====未检测到马娘数据库文件，正在开始创建文件和更新信息====
[2024-03-01 11:42:35,175 hoshino] INFO: 正在更新数据至最新，请耐心等待，预计用时1分钟
[2024-03-01 11:42:35,192 nonebot] ERROR: Failed to import "hoshino.modules.uma_plugin.umamusume_news", error: [WinError 5] 拒绝访问。: './res/img\\umamusume\\base_data/.git\\objects\\00\\44cad268d72a9630c5d0742326dfd73e06529d'
[2024-03-01 11:42:35,195 nonebot] ERROR: [WinError 5] 拒绝访问。: './res/img\\umamusume\\base_data/.git\\objects\\00\\44cad268d72a9630c5d0742326dfd73e06529d'

**Reply by lTinchl** [reply]
挂不挂vpn，设不设置代理都这样

**Reply by azmiao** [reply]
话说你hoshino下的res文件夹在吗，检查下路径在不在。你是原版hoshino吗，原版需要根据hoshino放置res文件夹

**Reply by lTinchl** [reply]
用你教程网站给的精简版提示
[2024-03-01 11:59:44,754 hoshino] ERROR: ====马娘数据库更新失败：Cmd('git') failed due to: exit code(128)
  cmdline: git clone -v --branch=main https://ghproxy.com/https://github.com/azmiao/uma_info_data.git ./res/img\umamusume\base_data/
  stderr: 'Cloning into './res/img\umamusume\base_data'...
fatal: unable to access 'https://ghproxy.com/https://github.com/azmiao/uma_info_data.git/': OpenSSL SSL_connect: Connection was reset in connection to ghproxy.com:443

![{1C0AFA45-E353-4b69-83B4-5300DD2D5D7E}](https://github.com/azmiao/uma_plugin/assets/105836889/17af637b-0a67-4538-aab0-77bee7b559e9)是否是这个的问题？

**Reply by azmiao** [reply]
是的，你可以尝试手动删除uma_plugin文件夹重新git clone一份

**Reply by lTinchl** [reply]
终于弄好了，感谢大佬
![45C9AAACEB8FE1AC1C53C233AF942C92](https://github.com/azmiao/uma_plugin/assets/105836889/afb1178c-12bd-408b-8387-a883d2ee5a3d)

**Reply by lTinchl** [reply]
寄！现在Code:不是237就是47，要不然就是用一会就风控

**Reply by azmiao** [reply]
目前QQ的检测很严格，尤其是新号，只能说功能尽量一点一点开，用一段时间再加功能，一开始发消息不要太多太快。

code237可以尝试修改bot密码，然后用gocqhttp启动，不要开启hoshino，等待挂了24小时后，再启动hoshino链接。

code45请确保qsign链接上gocq。

另外这里有减少风控概率的教程，请按需使用，但目前24年了不一定有咲，仅供参考https://www.594594.xyz/2023/09/25/qqbot_anti_risk/

**Reply by lTinchl** [reply]
我在网上搜到[LLOneBot](https://github.com/LLOneBot/LLOneBot),这个是否可以当go-cqhttp来用?

**Reply by azmiao** [reply]
理论上当然可行，不过我还没用过（，你可以试试

**Reply by azmiao** [reply]
liteloader我都很久没更新了，这个插件也是才看到

**Reply by lTinchl** [reply, marked as answer]
部署成功了 大佬
![image](https://github.com/azmiao/uma_plugin/assets/105836889/24ce721a-8866-4b48-af28-5e3298857223)

**Reply by azmiao** [reply]
可以可以

**Reply by lTinchl** [reply]
还得是你写的bot，没你的bot每次查节奏榜都得点开炮姐的视频看，谢谢分享
![image](https://github.com/azmiao/uma_plugin/assets/105836889/a11e18e5-f1dd-44a0-b3f9-f89c685029de)

**Reply by azmiao** [reply]
@_@
